vector/src/main/scala/geotrellis/vector/io/json/Style.scala

```diff
@@ -0,0 +1,86 @@
+package geotrellis.vector.io.json
+
+import spray.json._
+import scala.collection.mutable
+
+case class Style(
+  strokeColor: Option[String],
+  strokeWidth: Option[Int],
```

Comment:
This can be Unit, which is "1", "1.5", "5%"
http://wiki.openstreetmap.org/wiki/Geojson_CSS
http://www.w3.org/TR/SVG/painting.html#StrokeWidthProperty